### ISSUE TITLE
Some Icons

### ISSUE THREAD

**Issue by frosticons** [dkanada/frost#486]
Hey, I wanted to contribute [two icons](https://github.com/dkanada/frost/files/6687763/icons.zip), one for youtube music vanced and one for the simple dialer app (fdroid).
Now, I don't know how git works so I just wanted to ask if someone else could upload them if they are found suitable.

**Comment by baitmooth** [comment]
Thanks for the icons. If you could also add the activity names like described in https://github.com/dkanada/frost#icon-requests, I will try and include them.

**Comment by frosticons** [comment]
Dialer
\<item component="ComponentInfo{com.simplemobiletools.dialer/com.simplemobiletools.dialer.activities.SplashActivity.Purple}" drawable="dialer"/>

YT Music
\<item component="ComponentInfo{com.vanced.android.apps.youtube.music/com.google.android.apps.youtube.music.activities.MusicActivity}" drawable="ytmusic"/>

This is the information I got from Turtl, though I am not sure whether the activity name for the dialer app is of relevance. The dialer app offers different colors for its icon and the activity includes said color ("Purple"). Does that mean only the purple version of the icon will be affected?

Anyways, thanks for trying to include my icons :)

**Comment by frosticons** [comment]
Here's also an icon for Linguee, though this one I didn't create myself, I pretty much just copied it from https://www.linguee.com/.
I don't know whether that's legal, but if it is, this is the [icon](https://github.com/dkanada/frost/files/6711796/linguee.zip) and the data:

\
\<item component="ComponentInfo{com.linguee.linguee/com.linguee.linguee.MainActivity}" drawable="linguee"/>

**Comment by baitmooth** [comment]
> This is the information I got from Turtl, though I am not sure whether the activity name for the dialer app is of relevance. The dialer app offers different colors for its icon and the activity includes said color ("Purple"). Does that mean only the purple version of the icon will be affected?

Thats beyond my understanding how those icons work and I haven't seen it before. If it is relevant @dkanada hopefully knows. I only drop by to draw icons.

**Comment by baitmooth** [comment]
> Here's also an icon for Linguee, though this one I didn't create myself, I pretty much just copied it from https://www.linguee.com/.
> I don't know whether that's legal, but if it is, this is the icon

I also don't know that. And often my understanding of real legal stuff is quite naive. If you drew some of the lines and don't claim to own linguee it should be fine..

**Comment by dkanada** [comment]
Since it says purple, you probably need to include the activity names for every single color it supports to make sure they all have the proper icon from this pack.

**Comment by baitmooth** [comment]
I scaled your icons a little bit. youtubemusic was too big and linguee, and dial slightly too small. I hope thats ok for you, @frosticons .
besides that those are all three nice icons. good work.

The other thing is about the dialer app, I took a look at the relevant file and the icon is already present, but only for 'orange'. All other apps in this set have the same feature and have all the colors set to point to the same icon.

In short: If you change to 'orange' you get the frosticon icon right now.

I guess that @dkanada won't accept an unique icon for every color ;)
